packages/compass-home/src/components/home.tsx

```diff
@@ -267,7 +267,7 @@ function Home({ appName }: { appName: string }): React.ReactElement | null {
     return (
       <div className={homeViewStyles} data-test-id="home-view">
         <div className={homePageStyles}>
-          <Connections />
+          <Connections appRegistry={appRegistry} />
```

Comment:
Sounds good, chatted offline a bit. For this case since it's only one function we're using appRegistry for we will instead pass the function and keep the interface smaller and avoid the context for now. Thanks for bringing this up